renovate.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema":"https://docs.renovatebot.com/renovate-schema.json",
+  "extends":[
+     "config:base"
+  ],
+  "timezone":"America/Los_Angeles",
+  "schedule":[
+     "after 3am on the 1st and 3rd day instance on Monday"
+  ],
+  "enabledManagers":[
+    "npm"
+ ],
+ "vulnerabilityAlerts": {
+   "labels": ["security"],
+   "schedule": ["at any time"]
+ },
+  "rangeStrategy":"replace",
+  "npm":{
+     "stabilityDays": 2
```

Comment:
nit: thoughts on extending this to a week? I don't have any data, but just thinking it'd provide more time for a fix release to be published. Looking at their docs, sounds like at a minimum we might want this set to `3`:

> npm packages less than 72 hours (3 days) old can be unpublished, which could result in a service impact if you have already updated to it. Set stabilityDays to 3 for npm packages to prevent relying on a package that can be removed from the registry